test/test_utils.py

```diff
@@ -0,0 +1,15 @@
+from utils import utils
+
+
+class TestUtils(object):
+    def test_str_to_boolean(self):
+        cases = [
+            ("True", True),
+            ("False", False)
+        ]
+        for s, gt in cases:
+            assert utils.str_to_boolean(s) == gt
```

Comment:
This function, also can prevent the wrong value of the parameter. For example, if we `call str_to_boolean("something wrong")`, it will return `false` by default. I am not sure if the test needs to cover this case.

Reply:
It should, but let's do it in later PRs.

Also the function should probably return True when passed "true", which is not the case now.